src/main/java/edu/ucsb/cs156/rec/entities/RecommendationRequest.java

```diff
@@ -25,6 +27,10 @@ public class RecommendationRequest {
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   private long id;
 
+  @ManyToOne
+  @JoinColumn(name = "user_id", referencedColumnName = "id", insertable = false, updatable = false)
+  private User user;
+
   private String professorName;
   private String professorEmail;
```

Comment:
```suggestion
  @ManyToOne
  @JoinColumn(name = "user_id", referencedColumnName = "id", insertable = false, updatable = false)
  private User professor;
```

Then for RecommendationRequest r you can access:

```
    r.getProfessor().getName()
    r.getProfessor().getEmail()
```